bob-apps/bin/bobp.rs

```diff
@@ -583,6 +590,10 @@ fn print_periodic_stat(
             get_speed_values.push(get_spd);
         }
         if print_exist {
+            if multiline {
```

Comment:
Inconsistent prefix processing. This will not work with `put` + `exist` combination. I suggest to add a boolean flag, that keeps track of whether the first line was printed. With that flag you can check it at the beggining of every block (`print_put`, `print_get` and `print_exist`) and print the prefix if needed